resources/public/api/conf/2.0/application.raml

```diff
@@ -33,6 +33,19 @@ documentation:
 
 '/individuals/business/property':
   /uk:
+    /furnished-holiday-lettings:
+      /{nino}:
+        uriParameters:
+          nino:
+            description: National Insurance number, in the format AA999999A.
+            type: string
+            example: TC663795B
+        /{taxYear}:
+          uriParameters:
+            taxYear:
+              description: 'The tax year to which the data applies. For example: 2022-23. The start year and end year must not span two tax years. The minimum tax year is 2022-23. No gaps are allowed, for example, 2022-24 is not valid. (The minimum tax year in Sandbox is 2021-22.)'
```

Comment:
Unusually, the year 2022-23 would not be valid for this endpoint. Since it's a historic endpoint it will only accept year ranges from between 2017-18 and 2021-22.  Could we edit the description to reflect this?

Reply:
This has now been updated accordingly